ansible_collections/arista/avd/docs/release-notes/5.x.x.md

```diff
@@ -41,6 +41,56 @@ for details.
 
 TODO: add level4 sections for breaking changes outlining the change and referring to porting guide for migration.
 
+### Breaking or behavioral changes in eos_validate_state
+
+Starting AVD 5.0.0, `eos_validate_state` uses ANTA as backend, which will change the default behavior (same as setting `use_anta: true` in the latest AVD 4.x.x versions).
+If you have not been using `use_anta: true` in latest versions of AVD 4.x.x, please go through the following sections for details on the changes.
+
+Please report any issues on the GitHub [discussions board](https://github.com/aristanetworks/avd/discussions).
+
+Breaking changes may require modifications to the inventory, input variables or playbooks. See the [Porting guide for AVD 5.x.x](../porting-guides/5.x.x.md)
+for details.
+
+#### Loose mode to ignore playbook error is no longer supported
+
+No porting guide entry.
```

Comment:
Instead maybe add a small description on how errors are handled in the new integration? Like an unreachable device.